### PR TITLE
fix: update insight prompt

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

Screen Shot 2022-09-09 at 2 27 26 PM

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/insights/Insight.tsx [hunk 1]
```diff
@@ -301,12 +302,14 @@ export function Insight({ insightId }: { insightId: InsightShortId | 'new' }): J
...
             />
 
-            {actorOnEventsQueryingEnabled ? (
+            {actorOnEventsQueryingEnabled && alertVisible ? (
                 <div className="mb-4">
-                    <AlertMessage type="info">
-                        Results looking different? We've changed how insights are calculated. Read more about this{' '}
-                        <Link to={`https://posthog.com/docs/how-posthog-works/queries`}>here</Link>. Reach out to us if
-                        you see something unexpected{' '}
+                    <AlertMessage type="info" onClose={() => setAlertVisible(false)}>
```
Comment (rcmarron) [inline frontend/src/scenes/insights/Insight.tsx:307]: This is going to re-appear on page refresh. We probably want it to go into local storage, so once a users dismisses it, it doesn't come back. Give me one min, I can add a `isCloseSticky` prop to AlertMessage.

Comment (EDsCODE) [reply frontend/src/scenes/insights/Insight.tsx:307]: Oh i thought we want it to come back on refresh

Comment (EDsCODE) [reply frontend/src/scenes/insights/Insight.tsx:307]: Oh you meant "sticky" as in it doesn't return 🤦🏻

Comment (rcmarron) [reply frontend/src/scenes/insights/Insight.tsx:307]: Hmm, that feels annoying to me. I'd imagine this is the type of thing that most users don't care about, and the ones that do care only need to read once. Is that how you're thinking about it?

Comment (EDsCODE) [reply frontend/src/scenes/insights/Insight.tsx:307]: Yeah I agree, I wasn't thinking too much of it when I first read your suggestion and I confused myself

Comment (EDsCODE) [reply frontend/src/scenes/insights/Insight.tsx:307]: Are you adding the prop

Comment (rcmarron) [reply frontend/src/scenes/insights/Insight.tsx:307]: Had a chat in Slack on this and we landed on just making it not-dismissible now.

**Review by rcmarron — APPROVED** [review]

**Pull request merged**